### ISSUE TITLE
Strip colours (etc.) from wrangler logs output

### ISSUE THREAD

**Issue by Cherry** [cloudflare/workers-sdk#4492]
### Describe the solution

The new wrangler logs are great for debug, but often include contents that make them really hard to parse. Example:

![image](https://github.com/cloudflare/workers-sdk/assets/856748/10d5c6f9-ab11-4a7c-8b0e-bebefccda3f9)

```
--- 2023-11-22T16:29:48.163Z log
 ⛅️ wrangler 0.0.0-08ae28c6
[38;2;255;136;0m---------------------------[39m
```

```
--- 2023-11-22T16:29:48.602Z warn
[33m▲ [43;33m[[43;30mWARNING[43;33m][0m [1mkj/async-io-win32.c++:982: warning: Bind address resolved to multiple addresses.  Only the first address will be used.  If this is incorrect, specify the address numerically.  This may be fixed in the future.; addrs[0].toString() = 127.0.0.1:62926[0m
```
[wrangler-2023-11-22_16-29-47_041.log](https://github.com/cloudflare/workers-sdk/files/13442711/wrangler-2023-11-22_16-29-47_041.log)

It would be ideal if these ansi colours and other things are removed from the logs output.

**Comment by penalosa** [comment]
We should probably leverage `strip-ansi` here, which is already a Wrangler dependency

**Comment by penalosa** [comment]
@claude implement this feature based on the issue description. Make sure to use the `strip-ansi` dependency. You should primarily need to modify the `appendToDebugLogFile()` function in the `packages/wrangler/src/utils/log-file.ts` file.

**Comment by penalosa** [comment]
@claude could you write a test for this?
